settings/gpapi.setting.php

```diff
@@ -13,4 +13,17 @@
     'is_domain' => 1,
     'is_contact' => 0,
   ),
+  'gpapi_gender_to_prefix_map' => array(
+    'group_name' => 'gpapi',
+    'group' => 'gpapi',
+    'name' => 'gpapi_gender_to_prefix_map',
+    'type' => 'Array',
```

Comment:
Civi's dev docs [discourage use of the array type](https://docs.civicrm.org/dev/en/latest/framework/setting/#supported-properties). I think `String` with `serialize` set to `CRM_Core_DAO::SERIALIZE_JSON` should work here.